modules/460-log-shipper/crds/doc-ru-cluster-log-destination.yaml

```
@@ -56,6 +56,8 @@ spec:
                               description: Закодированный в Base64 пароль для ключа.
                         verifyHostname:
                           description: Проверка соответствия имени удаленного хоста и имени, указанного в TLS-сертификате удаленного хоста.
+                        verifyCertificate:
+                          description: Проверка TLS-сертификата удаленного хоста. Сертификат не будет проверен на наличие в списках отозванных сертификатов (Certificate Revocation Lists).
```

```suggestion
                          description: |
                            Проверка TLS-сертификата удаленного хоста.
                            
                            Если установлен в `true`, то сертификат будет проверен по списку отозванных сертификатов (Certificate Revocation Lists).
```

@MariaMakeeva Any comments?

---

done